app/src/main/java/firebasebarcelona/firepadel/ui/courts/view/CourtListFragment.java

```diff
@@ -141,7 +141,7 @@ public void onAuthStateChanged(@NonNull FirebaseAuth auth) {
         if (user != null) {
           saveLocalPlayer(user);
         } else {
-          Toast.makeText(getActivity(), "Auth fail", Toast.LENGTH_SHORT).show();
+          //TODO show some kind of error
```

Comment:
TODO don't work. We do not have methodoloy for TODO. 

I rather delete them, and write down the task to design error msg feedback than leave a TODO here.